src/test/java/com/artipie/api/RepositoryVerticleTest.java

```diff
@@ -35,17 +37,18 @@ class RepositoryVerticleTest {
     /**
      * Service port.
      */
-    private static final int PORT = 8089;
+    private static int port;
 
```

Comment:
@olenagerasimova Test crashed again.
The cause of it is test starts before http server starts to listen incommig port.
It is required to wait http server port get available.